packages/xod-project/test/fixtures/deep-bound-values-propagation.xodball.json

```diff
@@ -14,6 +14,7 @@
             "x": 138,
             "y": 16
           },
+          "arityLevel": 1,
```

Comment:
I’m sure we must not add any implied fields in xodballs:

1. The `arityLevel` equal to 1 makes no sense for non-variadic nodes, so should not be here
2. Unless increased it shouldn’t be there even for variadic nodes